### DISCUSSION TITLE
[Nextcloud] How does calendar integration work?

### DISCUSSION THREAD

**Discussion by unalignedcoder** [the-djmaze/snappymail#924] [General]
Hello, I am just trying SnappyMail within Nextcloud.

I was wondering how the integration with Calendar works?
I see I can attach Nextcloud files, which is great, but what about Calendar events?

Ideally I should be able to link to them, or pull information from them, or even creating them from a received message.

But I cannot find any of these functions, either from within SnappyMail new message, or from the Calendar event itself.

**Comment by the-djmaze** [comment]
SnappyMail can insert calendar events from mail into calendar.
But due to bugs in calendar the events can crash calendar.

**Reply by mirkoozekker** [reply]
How does this work. Whenever I receive an invitation there is an ics attached. I can watch it as textfile inside the browser or download it. No way to take it to calendar. So I always download the ics switch to calendar and import the file, which is quite annoying.

**Reply by the-djmaze** [reply]
?admin -> extensions -> Nextcloud has a setting to enable it (experimental)
But as i said, Nextcloud calendar will crash due to bugs in Nextcloud Calendar

**Reply by hanserasmus** [reply]
So I had enabled this now, it worked, I only tested it once. I am on Nextcloud 28.0.1, with SnappyMail version 2.33.0.

Nothing crashed.

**Reply by the-djmaze** [reply]
Good to hear that

**Comment by hanserasmus** [comment]
Can report back that I have tested about 15 or so calendar events, and NC did not crash once. So I suspect whatever bug there was, had been fixed. Sent from my iPhoneOn 23 Jan 2024, at 17:26, the-djmaze ***@***.***> wrote:﻿
Good to hear that

**Reply by the-djmaze** [reply]
I understand it works for you.
It does not work for me due to a simple issue with Kontact/Kmail.
This one problem is inside Sabre (used by Nextcloud) and still not fixed.

And there are some others why Nextcloud fails.

**Reply by the-djmaze** [reply]
https://github.com/nextcloud/calendar/issues/4684
